Setup/Product.wxs

```diff
@@ -599,14 +589,6 @@
               - The add-in feature state (!feature-state) for the Visual Studio version is installed (3 = INSTALLSTATE_LOCAL). See:
                 Conditional Statement Syntax (http://msdn.microsoft.com/en-us/library/aa368012.aspx)
             -->
-            <Custom Action="RemoveVS2005AddInCommands"
-                 After="RemoveFiles">Installed AND NOT UPGRADINGPRODUCTCODE AND VS2005INSTFOLDER
-                                     AND !VS2005FeatureId=3</Custom>
```

Comment:
> Do we need to uninstall addin for 2008 to be able to install 2010 one?
No

I mean if you user had already installed VS2005-VS2008 addin, shouldn't we remove it in any case because we do not support it anymore?